### PR TITLE
Use shared helper library from probe-rs

### PR DESCRIPTION
**Pull request by Tiwalun**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -35,4 +36,4 @@ anyhow = "1.0.31"
 textwrap = "0.12.1"
 
 [patch.crates-io]
-probe-rs = { git = "https://github.com/probe-rs/probe-rs" }
+probe-rs = { version = "0.8.0", git = "https://github.com/probe-rs/probe-rs" }
```
Comment (Yatekii) [inline Cargo.toml:39]: This is not required anymore since it's in the regular dependencies now.

**Review by Yatekii — APPROVED** [review]
LGTM. Thanks!

I added a minor nitpick :)

File: Cargo.toml (hunk 1 above)
Comment (Tiwalun) [reply Cargo.toml:39]: I kept it because of the `probe-rs-rtt` dependency, otherwise there are two versions of `probe-rs` used.

Comment (Yatekii) [reply Cargo.toml:39]: Uhm probe-rs-rtt cannot use a different version? Otherwise it cannot interop which it does? I don't understand ...

Comment (Tiwalun) [reply Cargo.toml:39]: Without the patch here, probe-rs-rtt uses the released version, but cargo-embed uses the git version. This then leads to a conflict. With the patch, both use the git version.

Comment (Yatekii) [reply Cargo.toml:39]: Ah sorry, confused me for a bit :) Thanks for elaborating!

**Pull request merged**